docs/src/configuring-zed.md

```diff
@@ -3276,11 +3276,11 @@ Run the `theme selector: toggle` action in the command palette to see a current
   "default_height": 320,
   "default_model": {
     "provider": "zed.dev",
-    "model": "claude-3-7-sonnet-latest"
+    "model": "claude-4-sonnet"
   },
   "editor_model": {
     "provider": "zed.dev",
-    "model": "claude-3-7-sonnet-latest"
+    "model": "claude-4-sonnet"
```

Comment:
I know it's not the main goal of this PR, but it seems that the documentation and the default config file in `default.json` are outdated: there's no longer any `editor_model` setting. Looks like that main model is now always used for file edits as well. :slightly_smiling_face: